src/Monitoring/Sdk/Microsoft.DotNet.Monitoring.Sdk.csproj

```diff
@@ -12,7 +12,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.KeyVault" />
+    <PackageReference Include="Azure.Identity" />
+    <PackageReference Include="Azure.Security.KeyVault.Secrets" />
+    <!-- The version here had to be downgraded because the AzDo build wouldn't work with the one in Directory.Packages.Props 
+         More information can be seen at https://github.com/dotnet/arcade/issues/11953 -->
```

Comment:
Since this will end up referencing an issue that will be closed when this work is complete, I would just give an explanation about what you found in this comment. Something like "This version override needs to match the minimum version needed by Azure.Identity".